src/systems/systems/src/hydraulic/linear_actuator.rs

```diff
@@ -53,6 +56,268 @@ pub enum LinearActuatorMode {
+    total_volume_to_actuator: Volume,
+}
+impl LowPressureAccumulator {
+    const MEAN_ACCUMULATOR_PRESSURE_PSI: f64 = 800.;
+    const STDEV_ACCUMULATOR_PRESSURE_PSI: f64 = 200.;
+
+    const MAX_ACCUMULATOR_PRESSURE_PSI: f64 = 1300.;
+
+    const PRESSURE_TIME_CONSTANT: Duration = Duration::from_millis(1000);
+
+    const REFILL_FLOW_GALLON_PER_S: f64 = 0.05;
+
+    fn new() -> Self {
+        let init_pressure_psi = random_from_normal_distribution(
+            Self::MEAN_ACCUMULATOR_PRESSURE_PSI,
+            Self::STDEV_ACCUMULATOR_PRESSURE_PSI,
+        );
+
+        Self {
+            pressure: LowPassFilter::<Pressure>::new_with_init_value(
+                Self::PRESSURE_TIME_CONSTANT,
+                Pressure::new::<psi>(init_pressure_psi),
+            ),
+            total_volume_to_actuator: Volume::default(),
+        }
+    }
+
+    fn update(
+        &mut self,
+        context: &UpdateContext,
+        input_pressure: Pressure,
+        controller: &impl ElectroHydrostaticPowered,
+    ) {
+        let mut new_pressure =
+            if input_pressure > self.pressure.output() && controller.should_open_refill_valve() {
+                Pressure::new::<psi>(1800.)
+            } else {
+                self.pressure.output()
+            };
+
+        new_pressure = new_pressure
+            .min(Pressure::new::<psi>(Self::MAX_ACCUMULATOR_PRESSURE_PSI))
+            .max(Pressure::new::<psi>(0.));
+
+        let refill_flow_present = controller.should_open_refill_valve()
+            && (new_pressure - Pressure::new::<psi>(50.)) > self.pressure.output();
+
+        self.update_flow(context, refill_flow_present);
+
+        self.pressure.update(context.delta(), new_pressure);
+    }
+
+    fn update_flow(&mut self, context: &UpdateContext, refill_flow_present: bool) {
+        if refill_flow_present {
+            self.total_volume_to_actuator +=
+                VolumeRate::new::<gallon_per_second>(Self::REFILL_FLOW_GALLON_PER_S)
+                    * context.delta_as_time();
+        }
+    }
+
+    fn pressure(&self) -> Pressure {
+        self.pressure.output()
+    }
+
+    #[cfg(test)]
+    pub fn empty_accumulator(&mut self) {
+        self.pressure.reset(Pressure::default());
+    }
+}
+impl Actuator for LowPressureAccumulator {
+    fn used_volume(&self) -> Volume {
+        self.total_volume_to_actuator
+    }
+
+    fn reservoir_return(&self) -> Volume {
+        Volume::default()
+    }
+
+    fn reset_volumes(&mut self) {
+        self.total_volume_to_actuator = Volume::new::<gallon>(0.);
+    }
+}
+
+pub trait ElectroHydrostaticPowered {
+    fn should_open_refill_valve(&self) -> bool {
+        false
+    }
+
+    fn should_activate_electrical_mode(&self) -> bool {
+        false
+    }
+}
+
+#[derive(PartialEq, Copy, Clone)]
+pub enum ElectroHydrostaticActuatorType {
+    EHA,  // Can only run on electric mode or is damping
+    EBHA, // Can run in electric or from aircraft hydraulic pressure
```

Comment:
The comments are excellent, so please leave them in, but I suggest we use the full names for these, rather than the abbreviations.

Reply:
Ends up with a complicated enigmatic name :D